lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -148,6 +144,26 @@ public function getDefaultLength(AbstractPlatform $platform)
      */
     abstract public function getName();
 
+    private static function getTypeRegistry() : TypeRegistry
+    {
+        if (self::$typeRegistry === null) {
+            return self::$typeRegistry = self::createTypeRegistry();
```

Comment:
what about removing `return` on this, single return point on method should be cleaner